### ISSUE TITLE
allowParams* reports invalid configuration

### ISSUE THREAD

**Issue by eithed** [spaze/phpstan-disallowed-calls#153]
Trying to use any of the `allowParams*` configuration parameters results in `Invalid configuration:` error; using the configuration from examples:

```
parameters:
	disallowedFunctionCalls:
		-
			method: 'PotentiallyDangerous\Logger::log()'
			message: 'use our own logger instead'
			allowIn:
				- path/to/some/file-*.php
				- tests/*.test.php
			allowParamsInAllowed:
				-
					position: 1
					name: 'message'
					value: 'foo'
				-
					position: 2
					name: 'alert'
					value: true
			allowParamsAnywhere:
				-
					position: 2
					name: 'alert'
					value: true
```

results in:

```
Invalid configuration:
The item 'parameters › disallowedFunctionCalls › 18 › allowParamsInAllowed › 0' expects to be string, array given.
Invalid configuration:
The item 'parameters › disallowedFunctionCalls › 18 › allowParamsInAllowed › 1' expects to be string, array given.
Invalid configuration:
The item 'parameters › disallowedFunctionCalls › 18 › allowParamsInAllowed › 0' expects to be int|string|bool, array given.
Invalid configuration:
The item 'parameters › disallowedFunctionCalls › 18 › allowParamsInAllowed › 1' expects to be int|string|bool, array given.
Invalid configuration:
The item 'parameters › disallowedFunctionCalls › 18 › allowParamsAnywhere › 0' expects to be string, array given.
Invalid configuration:
The item 'parameters › disallowedFunctionCalls › 18 › allowParamsAnywhere › 0' expects to be int|string|bool, array given.
```

Can you please advise?

**Comment by spaze** [comment]
Hi, this is a bug introduced in [2.11.0](https://github.com/spaze/phpstan-disallowed-calls/releases/tag/v2.11.0) when the named params support was introduced. Sorry for that, I'll fix it like now. Mean while the non-named params should work, if you'd like to carry on :-)

**Comment by spaze** [comment]
The fix was released in [2.11.1](https://github.com/spaze/phpstan-disallowed-calls/releases/tag/v2.11.1) now, should be ok for you, let me know if not (or even if yes 😊).

Again, thanks & sorry.

**Comment by eithed** [comment]
No worries - thank you for resolving this so quickly! Will proceed with testing what I've touched on in discussion and will get in touch if it's not working. Thanks again!
